mock/src/transaction.rs

```diff
@@ -190,6 +314,40 @@ impl MockTransaction {
     /// Consumes the mutable ref to the MockTransaction returning the structure
     /// by value.
     pub fn build(&mut self) -> Self {
+        let tx = TransactionRequest::new()
+            .from(self.from.address())
+            .to(self.to.clone().unwrap().address())
```

Comment:
I think this shouldn't fail if `to == None`, it should just set the `0x0` address

Reply:
Fixed in 6ce4c1d